tests/proxy/reliable_topic_test.py

```diff
@@ -0,0 +1,377 @@
+import os
+import time
+from datetime import datetime
+
+import hazelcast
+from hazelcast import ClientConfig
+from hazelcast.config import ReliableTopicConfig, TOPIC_OVERLOAD_POLICY, ClientProperties
+from hazelcast.exception import IllegalArgumentError, TopicOverflowError
+from hazelcast.proxy.reliable_topic import ReliableMessageListener
+from hazelcast.proxy.ringbuffer import OVERFLOW_POLICY_FAIL, OVERFLOW_POLICY_OVERWRITE
+from hazelcast.serialization.reliable_topic import ReliableTopicMessage
+from hazelcast.util import current_time_in_millis
+from tests.base import SingleMemberTestCase
+from tests.util import random_string, event_collector
+
+
+class _ReliableTopicTestException(BaseException):
+    pass
+
+
+class TestReliableMessageListener(ReliableMessageListener):
+    def __init__(self, collector):
+        self._collector = collector
+
+    def on_message(self, event):
+        self._collector(event)
+
+
+class TestReliableMessageListenerLossTolerant(ReliableMessageListener):
+    def __init__(self, collector):
+        self._collector = collector
+
+    def on_message(self, event):
+        self._collector(event)
+
+    def is_loss_tolerant(self):
+        return True
+
+
+class ReliableTopicTest(SingleMemberTestCase):
+    @classmethod
+    def configure_cluster(cls):
+        path = os.path.abspath(__file__)
+        dir_path = os.path.dirname(path)
+        with open(os.path.join(dir_path, "hazelcast_topic.xml")) as f:
+            return f.read()
+
+    def setUp(self):
+        config = ClientConfig()
+        config.set_property("hazelcast.serialization.input.returns.bytearray", True)
```

Comment:
As I mentioned before, you can remove this line after performing the small fix I mentioned here https://github.com/hazelcast/hazelcast-python-client/pull/206#issuecomment-628522527 (item 1)